include/bitbuffer.h

```diff
@@ -14,7 +14,7 @@
 
 #include <stdint.h>
 
-#define BITBUF_COLS 256 // Number of bytes in a column
+#define BITBUF_COLS 512 // Number of bytes in a column
```

Comment:
Make it 450 then and write a comment that m_bus needs this.

Reply:
fixed